adapter/list/list_test.go

```diff
@@ -63,9 +63,9 @@ func TestBasic(t *testing.T) {
 	}
 }
 
-func contains(s []string, e string) bool {
+func containsListEntryTemplate(s []string) bool {
```

Comment:
linter error: the method was only ever called with one string as `e`. `unparam` check.